tests/nhsd.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -euo pipefail
+./nhsd-backend/manage.sh
+#run again to check for idempotency
+./nhsd-backend/manage.sh
```

Comment:
I love (in an entirely non-ironic way) the fact that the test is just to do the same thing twice and check it doesn't blow up.

Reply:
I'd like to add some more assertions here. I'm thinking about pulling in https://github.com/sstephenson/bats to make tests better here.